repos/fdbt-site/src/pages/api/csvUpload.ts

```diff
@@ -198,15 +209,54 @@ export const faresTriangleDataMapper = (
     return mappedFareTriangle;
 };
 
+export const getNamesOfFareZones = (ticket: WithIds<SingleTicket> | WithIds<ReturnTicket>): string[] | undefined => {
+    let fareZoneNames;
+
+    if ('fareZones' in ticket) {
+        fareZoneNames = ticket.fareZones.map((fz) => fz.name);
+    } else if ('inboundFareZones' in ticket && 'outboundFareZones' in ticket) {
+        const combinedFareZones = ticket.outboundFareZones.concat(ticket.inboundFareZones);
+
+        const combinedNamesOnly = combinedFareZones.map((x) => x.name);
+
+        // get rid of duplicates
+        fareZoneNames = [...new Set(combinedNamesOnly)];
+    }
+
+    return fareZoneNames;
+};
+
+export const thereIsAFareStageNameMismatch = (fareTriangleData: UserFareStages, fareZoneNames: string[]): boolean => {
```

Comment:
Could we do it like this? 
```suggestion
export const thereIsAFareStageNameMismatch = (fareTriangleData: UserFareStages, fareZoneNames: string[]): boolean => fareTriangleData.fareStages.some((fs) => {
        return !fareZoneNames.includes(fs.stageName);
    });

```

Reply:
```suggestion
export const thereIsAFareStageNameMismatch = (fareTriangleData: UserFareStages, fareZoneNames: string[]): boolean =>
    fareTriangleData.fareStages.some((fs) => {
        return !fareZoneNames.includes(fs.stageName);
    });

```